csrc/scheduler/pointwise.cpp

```diff
@@ -700,41 +707,58 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams* pparams) {
       reference_tv->split(1, NamedScalar::getParallelDim(ParallelType::TIDx));
       // [outer | i-remainder, TIDx, Vect]
 
-      reference_tv->split(0, pparams->unroll_factor);
-      // [o-remainder, Unroll| i-remainder, TIDx, Vect]
+      if (pparams->unroll_factor_inner > 1) {
+        reference_tv->split(1, pparams->unroll_factor_inner);
+      }
+      // [outer| i-remainder, i-Unroll, TIDx, Vect]
+
+      if (pparams->unroll_factor_outer > 1) {
+        reference_tv->split(0, pparams->unroll_factor_outer);
+      }
+      // [o-remainder, o-Unroll| i-remainder, i-Unroll, TIDx, Vect]
```

Comment:
I'm a bit lost about the notation here. What's `o-Unroll | i-remainder`?

Reply:
`o` represents outer dim and `i` represents inner dim. `|` sperates inner dim and outer dim. So here `o-Unroll` represents `outer unroll` and `i-remainder` means `what is left in the inner dim after splitting out other domains, e.g. Vect, TIDx`

Reply:
ah, sorry I was totally not getting `|` part here. Now it reads clear to me.

Reply:
add some comments for clarity.
```
      // Here and in the following comments:
      // prefix [i] represents inner dimension
      // prefix [o] represents inner dimension
      // [|] separates the outer and inner dimensions
```